edx_arch_experiments/kafka_consumer/tests/test_bridge_use.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+from django.test import TestCase
+
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from confluent_kafka.schema_registry.avro import AvroSerializer, AvroDeserializer
+from opaque_keys.edx.keys import CourseKey
+from openedx_events.bridge.avro_attrs_bridge import AvroAttrsBridgeKafkaWrapper
+from openedx_events.learning.signals import COURSE_ENROLLMENT_CREATED
+from openedx_events.learning.data import CourseEnrollmentData, CourseData, UserPersonalData
+from unittest.mock import Mock
+
+class BridgeTest(TestCase):
+
+    def test_produce_and_consume_event_with_bridge(self):
+        my_bridge = AvroAttrsBridgeKafkaWrapper(COURSE_ENROLLMENT_CREATED)
+        mock_src = SchemaRegistryClient({'url': 'https://www.example.com'})
+        mock_src.register_schema = Mock(return_value=3)
+
+        serializer = AvroSerializer(schema_str=my_bridge.get_schema_str(),
```

Comment:
The bridge can create a schema_str from the signal by looking at its init_data attribute

Reply:
I wasn't clear that `get_schema_str()` already existed, and the bridge was already publicizing the schema. Glad that is already there.